RUFAS/simulation_engine.py

```diff
@@ -177,20 +184,40 @@ def _annual_simulation(self) -> None:
 
         self._run_post_annual_routines()
 
+    def annual_reset(self):
+        """
+        Description:
+            Resets all annual variables that require reset
+        """
+        self.field_manager.annual_update_routine()
+
+    def annual_mass_balance(self, time):
+        pass
```

Comment:
```suggestion
    def annual_mass_balance(self, time) -> None:
        pass
```
Since this method isn't doing anything, maybe it would just be better to delete it? If it is going to be kept though, I think it would be good to either have an issue written specifically about addressing it or mention it specifically in the issue about refactoring State

Reply:
I don't know exactly what the out of having this placeholder function here, I'll create an issue about it.